keyboards/nckiibs/vaneelaex/vaneelaex.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2020 noclew
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_ss_6x12( \
```

Comment:
This layout has a 2U spacebar in the center, but is otherwise ortho? 
If so, it may be better to use `LAYOUT_ortho_6x12_1x2uC` for the layout name. 

(but I'm horrible at naming things.

Reply:
Oh, it is semi-staggered, so I just put ss. If it is not appropriate, please let me know!